src/test/java/cat/udl/eps/softarch/steps/UserStepDefs.java

```diff
@@ -0,0 +1,71 @@
+package cat.udl.eps.softarch.steps;
+
+import cat.udl.eps.softarch.Softarch1617Application;
+import cat.udl.eps.softarch.domain.User;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import cucumber.api.PendingException;
+import cucumber.api.java.Before;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootContextLoader;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.util.Objects;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+
+/**
+ * Created by jennifer on 4/10/16.
+ */
+@ContextConfiguration(
+        classes = {Softarch1617Application.class}, loader = SpringBootContextLoader.class
+)
+@DirtiesContext
+@RunWith(SpringRunner.class)
+@WebAppConfiguration
+
+public class UserStepDefs {
+    private static final Logger logger = LoggerFactory.getLogger(UserStepDefs.class);
+
+    @Autowired private WebApplicationContext wac;
+    private MockMvc mockMvc;
+    private ResultActions result;
+    private ObjectMapper mapper = new ObjectMapper();
+
+    private User user = new User();
+
+    @Before
+    public void setup() {
+        this.mockMvc = MockMvcBuilders
+                .webAppContextSetup(this.wac)
+                .build();
+    }
+
+    @When("^I create an user with name \"([^\"]*)\" and lastname \"([^\"]*)\"$")
```

Comment:
The creation of a user (or a buyer) should be done using the API, using the mockMvc object and performing an HTTP POST to /users (/buyers)